### DISCUSSION TITLE
Responsive Grid doesn't work on Mobile devices

### DISCUSSION THREAD

**Discussion by sazzer** [fomantic/Fomantic-UI#1061] [Collections (form, grid, menu, message, table, ...)] [answered]
# Bug Report

I'm using a Mobile Responsive Grid, so that when my page is rendered on a large screen I get two columns side-by-side and on a small screen they are stacked. This *seems* to work, but only when rendering in "Desktop Mode" (Whatever that means)

## Steps to reproduce
Exact markup that I'm using:
```
        <div class="ui mobile reversed stackable grid container">
            <div class="four wide column">
                {% block menu %}{% endblock %}
            </div>
            <div class="twelve wide column">
                {% block form %}{% endblock %}
            </div>
        </div>
```

## Expected result
When rendered in a normal desktop browser:
![image](https://user-images.githubusercontent.com/66072/66206693-be1d5b80-e6a8-11e9-9352-538a5b0d3a61.png)

When rendered in Desktop Chrome, in Responsive Mode set to Desktop:

![image](https://user-images.githubusercontent.com/66072/66206786-063c7e00-e6a9-11e9-8fce-354292d28118.png)

## Actual result

When rendered in Desktop Chrome, in Responsive Mode set to Mobile:

![image](https://user-images.githubusercontent.com/66072/66206771-ff157000-e6a8-11e9-852b-796e5eb14c46.png)

Rendering this in Android Chrome on my actual phone *always* looks like this, regardless of whether Desktop Mode is ticked or not.

## Testcase
The exact code used is above. Jsfiddle doesn't seem to work great to replicate mobile-mode issues though, since it doesn't work in that mode.

## Version
Fomantic UI - 2.7.8

**Comment by lubber-de** [comment]
> Jsfiddle doesn't seem to work great to replicate mobile-mode issues though, since it doesn't work in that mode.

It does, just append "/show"  to the jsfiddle URL

**Comment by lubber-de** [comment]
Please provide a jsfiddle with your content, because i believe your issue is related to the used content inside the columns.
Using only your little code above results in a fine working behavior:

https://jsfiddle.net/btc2rue0/
https://jsfiddle.net/btc2rue0/show (for mobile view)

**Comment by sazzer** [comment]
Ok - the "/show" bit I didn't know about, but very worth knowing :)

Curiously, the *exact* markup that I'm using in reality - I literally copied it out of Chrome Developer Tools - works fine in jsfiddle but not in my app - both when running in Chrome Desktop using Mobile Mode. The Jsfiddle also doesn't work on my actual phone, so it *does* reproduce the issue somewhat. 

That markup is at https://jsfiddle.net/jrLkwmu2/2 (And https://jsfiddle.net/jrLkwmu2/2/show for mobile view)

Cheers

**Comment by sazzer** [comment]
Screenshot from my phone of the Jsfiddle page:
![image](https://user-images.githubusercontent.com/66072/66208225-80bacd00-e6ac-11e9-8cd5-5714d35834b8.png)

**Comment by lubber-de** [comment, marked as answer]
Basic solution: Use `five wide column` and `eleven wide column` instead, because your menu content does not fit. See https://jsfiddle.net/58op7kc6/

Other than that  your fiddle worked completely fine for me on my mobile phone.
Your mobile browser is probably configured to force Desktop Mode. Only then, i can reproduce your issue.

**Comment by lubber-de** [comment]
Closing because reason of behavior was found and solution provided.
